### ISSUE TITLE
Port change is not reflected in usage of Syncthing (manual restart is needed)

### ISSUE THREAD

**Issue by ineiti** [xor-gate/syncthing-macosx#198]
On my mac I have two syncthing instances, one for each user. So I changed the port of one of the instances in order for them to run in parallel. Now the following happens for the syncthing with the changed port:

- it seems to me that the status icon follows the syncthing of the other user with the default port
- when I click on the icon and choose `Open`, it opens the syncthing with the default port. Same thing if I click the icon, choose `Preferences` and then `Open Syncthing in Browser`

**Comment by xor-gate** [comment]
you need to click on the icon -> Preferences -> enter the URI -> click test. Then close syncthing and restart it. Then the non default port should be used. Keep in mind you should also set the API key based on the instance.

**Comment by ineiti** [comment]
OK, seems I didn't quit and start `Syncthing` again.

**Comment by xor-gate** [comment]
Reopened because we should at least document this behaviour. And probably we should fix it so it is live changed and it doesn't need restarted. For my own administration: this behaviour is probably due to the default functionality and syncthing daemon background process are not synced when updating. Then the background process should be stopped/killed and then started again after the change and given the new setting.

**Comment by ineiti** [comment]
There might be other configuration changes that need to be reloaded, no? So perhaps for every configuration change, do this reloading of the daemon.

**Comment by xor-gate** [comment]
Exactly, like the environment variables and the hidden commandline arguments.

**Comment by DarthJahus** [comment]
Seems like entering 0.0.0.0:22001, for example, makes Syncthing say semicolon is not expected, and it continues on using the default 22000. But if you enter tcp://0.0.0.0:22001, it correctly changes and listens on 22001.
